modules/ROOT/pages/notifications/index.adoc

```diff
@@ -1,35 +1,203 @@
+The notification object consists of the following fields:
 
-The following categories exist, ordered by severity:
+.Neo4j notification object
+[cols="<1s,<4"]
+|===
+|Neo4j code
+a|The Neo4j status code in the form of `Neo.ClientNotification.[SubType].[Name]`.
+|Title
+a|The title of the notification.
+|Description
+a|The description of the Neo4j status code.
+|Severity level
+a|The severity can be one of the following:
+
+- `WARNING`: There might be a problem with your query. Please, take a look.
+- `INFORMATION`: The query is correct, but this information can still be useful.
+|Category
+a|The category of the notification.
+|Position
+a|The position, given by row and column, where the notification is relevant in the query text.
+|===
+
+
+[role=label--version-5.23]
+[[gqlstatus-notification-object]]
+== GQL-status notification object
+
+In the GQL-status object framework, notifications are an implementation-defined subset of the GQL-status objects that cover informational notes and warnings, but not errors, `SUCCESS`, `NO DATA`, or `OMITTED RESULT`.
+For the latter, see <<general-codes-for-success, General codes for success>>. +
+
+In GQL, the execution of a query from user to server always has an outcome, called the _execution outcome_.
+The execution outcome is a list of GQL-status objects, ordered by the following precedence, where the first object in the list is the primary GQL-status object:
+
+. `NO DATA` has precedence over `WARNING`.
+. `WARNING` has precedence over the `SUCCESSFUL COMPLETION` subclass.
+. `SUCCESSFUL COMPLETION` subclass has precedence over `INFORMATIONAL`.
+. `INFORMATIONAL` is the condition with the least precedence.
+
+The primary GQL-status object describes the condition with the greatest precedence and is always present.
+All other GQL-status objects in the list are additional GQL-status objects.
+
+The GQL-status object also includes the Neo4j-specific information, such as the severity level and the classification of the notification, which can be used for filtering.
+For more information about notification grouping and filtering, see <<notification-grouping-and-filtering>>.
+
+Each GQL-status object consists of the following fields:
+
+.GQLSTATUS notification object
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+a| A 5-character string that is the concatenation of a 2-character class code followed by a 3-character subclass code.
+|StatusDescription
+a| A human-readable description of the GQLSTATUS, which consists of a condition, a subcondition, and a description.
+The condition and subcondition are textual representations of the class and subclass codes, respectively.
+| DiagnosticRecord
+a| Extra information about the status, given as key-value pairs, both on the server and driver side.
+The full diagnostic record can be returned by the `getDiagnosticRecord()` method on the server side and the `diagnosticRecord()` methods on the driver side.
```

Comment:
```suggestion
To retrieve the full diagnostic record, you can use the `diagnosticRecord()` method on both the server and driver sides.
```

Reply:
Good suggestion, please go ahead and merge